samples/hydrogen-prepackaged/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>javascript-modules</artifactId>
+        <groupId>org.jahia.modules</groupId>
+        <version>0.7.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>javascript-modules-samples-hydrogen-prepackaged</artifactId>
```

Comment:
Any preferences for the `groupdId` / `artifactId` of the prepackaged content?
Should I use `org.jahia.samples` as `groupId` for both `hydrogen` and `hydrogen-prepackged`?
Looks like https://devtools.jahia.com/nexus/content/groups/public/org/jahia/samples/ already exists...